### PR TITLE
Add link sharing to the sharing explainer

### LINKED ISSUE #1629: Update the sharing tutorial to talk about links
Right now, the sharing tutorial only talk about adding ppl to a shared folder. We could think about adding a mention of link sharing? wdyt @sweetpea22 @serenaho ?

to be precise, I'm talking about this "tutorial"
image

### PR DESCRIPTION
**Pull request by Tbaut**
closes #1629

I added a test to check that the cross allows to dismiss the modal.
TODO:
- [x] Update the image to have a high res of it.

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

#### Theme
- [x] Components / elements inspected in light mode 
- [x] Components / elements inspected in dark mode

### CODE REVIEW

File: packages/files-ui/cypress/tests/sharing-explainer-spec.ts [hunk 1]
```diff
@@ -32,6 +33,30 @@ describe("Sharing Explainer", () => {
...
+
+      cy.web3Login()
+      navigationMenu.sharedNavButton().click()
+      sharingExplainerModal.body().should("be.visible")
+
+      // set up a spy for the POST response
+      cy.intercept("POST", "**/user/store").as("storePost").then(() => {
+
+        // dismiss the sharing explainer
+        sharingExplainerModal.closeButton().click()
```
Comment (asnaith) [inline packages/files-ui/cypress/tests/sharing-explainer-spec.ts:50]:
```suggestion
        sharingExplainerModal.closeButton().safeClick()
```

I tend to use `safeClick` for modal closures to remove any chance of "detached from DOM" issues or clicking before an event listener has been registered etc.

**Comment by asnaith** [comment]
Nice addition to the explainer, thanks for adding the new test too 🤩

I left one tiny suggestion and can approve once we get the high res asset :)

**Comment by Tbaut** [comment]
Updated the PR with the right image, it's now ready to go

File: packages/files-ui/cypress/tests/sharing-explainer-spec.ts (hunk 1 above)
Comment (Tbaut) [reply packages/files-ui/cypress/tests/sharing-explainer-spec.ts:50]: The safeclick is really meant for buttons, as we're checking for the `enabled` state, which is not available on this cross image. I went back to the click. We can figure it out if we ever have an issue. Safeclick remains kind of a hack that I'd rather use only when needed.

Comment (asnaith) [reply packages/files-ui/cypress/tests/sharing-explainer-spec.ts:50]: Ah ok 👍, just by looking at the test I thought it was a button. I guess to the user it is, at least functions as one, but if the element type is not actually `button` then yeah no need to use `safeClick`

**Review by asnaith — APPROVED** [review]

**Pull request merged**